.gitlab-ci.yml

```diff
@@ -0,0 +1,41 @@
+test-installer:
```

Comment:
What causes this particular test to be run as opposed to some other test (for example a future test would cause a Crucible update to run)?

Reply:
Gitlab has something called 'stages'. 
stages:
 - installer
 - update
 - performance
In this example, performance stage will execute only if installer and update stages succeeded.

You can also specify tests that run if a particular file is changed:
```
only: 
    refs:
      - master
    changes: 
      - crucible-installer.sh
      - bin/crucible
```